CHANGELOG.md

```diff
@@ -10,4 +10,6 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Fixed
 
+ - Update demo setup to be include ubi and ecommerce data sets and run in OS 3.1 ([#9](https://github.com/opensearch-project/search-relevance/pull/9))
```

Comment:
this should be a github issue, not PR. If you have one in o19s I think it can be transferred by @wrigleyDan or @epugh, because in order to transfer issue you need to be a maintainer in both source and destination repos. Otherwise we need to create new issue in https://github.com/opensearch-project/search-relevance, can be something generic/meta around preparing for Demo